pkg/authorization/authorizer/authorizer.go

```diff
@@ -35,6 +35,8 @@ type AuthorizationAttributes interface {
 	GetResourceName() string
 	// GetRequestAttributes is of type interface{} because different verbs and different Authorizer/AuthorizationAttributeBuilder pairs may have different contract requirements
 	GetRequestAttributes() interface{}
+	// GetRemainingUrl returns the steps that are present after parsing out the known steps.  Known steps commonly include prefixes and resource related steps.
+	GetRemainingUrl() []string
```

Comment:
do we care about remaining url parts, or non-resource url parts? depends on how these get used in combination with resource/resourcename